main/utils/highScores.c

```diff
@@ -0,0 +1,97 @@
+#include "highScores.h"
+
+#include "esp_log.h"
+#include "macros.h"
+#include "../../components/hdw-nvs/include/hdw-nvs.h"
```

Comment:
I think you can get away with 
```suggestion
#include "hdw-nvs.h"
```